bfkrickard/MusicGame
# Home screen setup

**Summary**:
- created initial project structure for containers and components

**Updates:**
- setup home container
   - initially will load the `LoginController`
- setup initial theme
   - for when we want to customize our theme globally later
- install [react-native-paper](https://callstack.github.io/react-native-paper/)
    - UI component library 
- setup `PlayContainer`
   - this is the container that has the choice selection and user(s) list
   - currently not accessible because we need to add routing/navigation feature
- added `jsconfig.json` for alias
  - so we can import with things like `AppContainers` instead of `./src/containers/other/path`
- created `iconButton.js` component for clickable answer
   - **Incomplete**
   - this button will be used for the clickable multiple choice section
   - there is a property `active` if set to `true` it will show a checked icon and button will have `active` style

**Testing**:
- pull this branch locally: *(gotta make sure you have no pending changes locally before switching branches)*
   1. `git fetch origin` - fetches any new updates on ALL remote branches
   2. `git checkout home-screen-setup` - pulls this branch 
- run `yarn install` - installs all the new dependencies
- run `yarn web` - runs the web app
- navigate to: http://localhost:19006/
- verify that you see this screen
  - ![image](https://user-images.githubusercontent.com/3317835/93736238-8f87e180-fb94-11ea-83f9-1cc78e1c9afa.png)

- Verify that the spotify button is disabled initially
- Enter a nonempty value to the name (test edge cases like just spaces or whatever else)
- Verify that on a nonempty name value, the spotify button becomes `active` and you can click it
